addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -57,6 +57,9 @@ export class OutOfFocusService {
             title: notificationTitle,
             type: "info",
         });
+        if (thread?.model === "discuss.channel") {
+            this._playSound();
```

Comment:
I would still prefer to send sound inside the `sendOdooNotification` and `sendNativeNotification`, because `sendNativeNotification` can fail and seems to be off when having the sound but actually not getting notified

Reply:
If the `sendNativeNotiication` fails, we do `sendOdooNotification`, so this will necessarily result to `_playSound()`.

Moving it to `sendNotification()` makes diffing these LOCs once, which is best because I put a conditional to affect only `discuss.channel` messages.

Reply:
This is a lot of API change in stable though, can't we have a play sound param to the other methods, which is true by default to keep the behavior, and only passed false from where we want to prevent it?

Reply:
ok i see what you mean.
Can we change the API of `async notify(message, channel)` to `async notify(message, channel, , sound = false)` I don't think that disabling/enabling something based on a hard-coded check of the model type inside of a basic function like `notify` is not a good idea and it is harder to maintain the code.

For example, I want to disable the sound when I trying to implement the mute stuff and found out it was coupled with another feature that causes my channel to always have sound. It can be really annoying.

Reply:
> This is a lot of API change in stable though, can't we have a play sound param to the other methods, which is true by default to keep the behavior, and only passed false from where we want to prevent it?

you were faster than me! xd